--- android/app/build.gradle	2023-03-01 00:00:00.000000000 +0000
+++ android/app/build.gradle	2023-03-01 00:00:00.000000000 +0000
@@ -42,7 +42,7 @@
     defaultConfig {
         applicationId "com.example.chunk_up"  // This may be different in your file
         // You can update the following values to match your application needs.
-        minSdkVersion 21
+        minSdkVersion 23  // Updated to 23 for google_mobile_ads compatibility
         targetSdkVersion flutter.targetSdkVersion
         versionCode flutterVersionCode.toInteger()
         versionName flutterVersionName